apps/cli/src/commands/run.ts

```diff
@@ -72,12 +72,12 @@ export default class Run extends Command {
         const epochDuration = flags["epoch-duration"];
         const listenPort = flags["listen-port"];
         const env: NodeJS.ProcessEnv = {
+            CARTESI_SNAPSHOT_DIR: "/usr/share/cartesi/snapshot",
```

Comment:
let's try to sort this list in alphabetical order

Reply:
sent a fixup